dvc/state.py

```diff
@@ -370,13 +370,12 @@ def save(self, path_info, checksum):
         assert path_info.scheme == "local"
         assert checksum is not None
 
-        path = fspath_py35(path_info)
-        assert os.path.exists(path)
+        assert os.path.exists(fspath_py35(path_info))
```

Comment:
The empty line above this looks superfluous now. A matter of taste for sure.

Reply:
@Suor I will change this once the changes are approved because there might be a change in this file so I can incorporate in that particular commit.